Lib/test/test_dataclasses.py

```diff
@@ -68,6 +68,26 @@ def test_field_repr(self):
 
         self.assertEqual(repr_output, expected_output)
 
+    def test_dataclass_params_repr(self):
```

Comment:
Can you add a comment here that says that even though this is testing an internal implementation detail, it's testing a feature we want to make sure is correctly implemented for the sake of dataclasses itself?